pkg/webhooks/subscription/subscription.go

```diff
@@ -0,0 +1,183 @@
+package subscription
+
+import (
+	"encoding/json"
+	"net/http"
+	"sync"
+
+	responsehelper "github.com/openshift/managed-cluster-validating-webhooks/pkg/helpers"
+	"github.com/openshift/managed-cluster-validating-webhooks/pkg/webhooks/utils"
+	"k8s.io/api/admission/v1beta1"
+	admissionregv1 "k8s.io/api/admissionregistration/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	admissionctl "sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+const (
+	WebhookName    string = "subscription-validation"
+	LoggingSubName string = "cluster-logging"
+	ESSubName      string = "elasticsearch-operator"
+	BlockedChannel string = "4.5"
```

Comment:
Should we make this an array? Only to make it easier to do last minute changes in case we need to block 4.6 channel as well

Reply:
I'd +1 this if we really expect to need to do this. My assumption is that this hook will be relatively short-lived.

Reply:
+1, it makes sense.

It seems 4.6 goes GA in 2 weeks. I don't know the timeline for the logging-team solving the open issues.

Reply:
I'm ok either way here. it's easy to implement. however, without being able to upgrade to 4.5, customers won't be able to get from 4.4 to 4.6 without ripping out the logging stack and reinstalling. and considering we're not sure if the issues are seen only when upgrading from 4.4 to 4.5 (or if they're also seen with fresh installs of 4.5), it's possible that ripping out 4.4 and reinstalling at 4.6 won't be problematic, if that's what the customer wants to do.

either way, I also hope this is short-lived and won't live long enough to see a need for this

Reply:
+1 to keeping a single blockedChannel for now